### PR TITLE
Update Error codes

### PR DESCRIPTION
**Pull request by jamiehannaford**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-4284f7e3ac/)

File: _reference/2016-04-29-error-codes.md [hunk 1]
```diff
@@ -14,57 +14,57 @@ topics:
...
 
-You attempted to run a container that mounts a volume from a restricted directory on the node by using a volume flag of the form `--volume /node/dir:/container/dir` or `-v /node/dir:/container/dir`. This action is not allowed in Carina because of security restrictions. 
+You attempted to run a container that mounts a volume from a restricted directory on the node by using a volume flag of the form `--volume /node/dir:/container/dir` or `-v /node/dir:/container/dir`. This action is not allowed in Carina because of security restrictions.
 
-To resolve this error, see the [Volumes]({{ site.baseurl }}/docs/concepts/docker-swarm-carina/#volumes) section in [Understanding how Carina uses Docker Swarm][docker-swarm-carina].
+To resolve this error, see the [Volumes]({{ site.baseurl }}/docs/concepts/docker-swarm-carina/#volumes) section in [Understanding how Carina uses Docker Swarm][docker-swarm-carina] and the [Kubernetes restrictions in Carina][kubernetes-swarm-carina].
 
 [docker-swarm-carina]: {{ site.baseurl }}/docs/concepts/docker-swarm-carina/
+[kubernetes-swarm-carina]: {{ site.baseurl }}/docs/concepts/kubernetes-101/#kubernetes-and-carina/
```
Comment (carolynvs) [inline _reference/2016-04-29-error-codes.md]: The trailing slash is preventing the bookmark from working.

**Comment by jamiehannaford** [comment]
@carolynvs Trailing slash removed 🐱

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-9325e5987d/)

**Pull request merged**